main.go

```diff
@@ -169,6 +171,47 @@ func filterBucketContent(elems s3.ListObjectsOutput, csvSuffix, processedFlagSuf
 	return unprocessed, processed, orphanFlags
 }
 
+func filterOutNewerWindows(objects s3.ListObjectsOutput, folderCount int) (olderObjects s3.ListObjectsOutput) {
```

Comment:
I wonder if you could improve `folderCount` naming, by reading the function definition it's unclear that it is the number of older windows to ignore. I don't think that's the greatest name, but maybe  `windowCountToIgnore` would be a bit better ?  ?

Reply:
Good point. I went for something even prettier: newestWindowCoundToIgore !

Reply:
Ahah, fine by me !